specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -0,0 +1,1726 @@
+          "SubscriptionQuotaAllocationRequests_List_Request_ForCompute": {
+            "$ref": "./examples/SubscriptionQuotaAllocationRequests/SubscriptionQuotaAllocationRequests_List_Request-Compute.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/MgIdInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionQuotaAllocationRequestList"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RequestState": {
+      "description": "Request status.",
+      "enum": [
+        "Accepted",
+        "Created",
+        "Invalid",
+        "Succeeded",
+        "Failed",
+        "InProgress",
+        "Canceled"
+      ],
+      "readOnly": true,
+      "type": "string",
+      "x-ms-enum": {
+        "name": "RequestState",
+        "modelAsString": true
+      }
+    },
+    "OperationsStatus": {
+      "description": "A long running process operation Status. Based on - https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#202-accepted-and-location-headers",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The operation status identifier. It should match what is used to GET the operation resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "It must match the last segment of the id field, and will typically be a GUID / system generated value."
+        },
+        "type": {
+          "description": "The type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "provisioningState": {
+              "$ref": "#/definitions/RequestState",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "AssignedToSubscription": {
+      "type": "object",
+      "description": "SubscriptionIds and quota allocated to subscriptions from the GroupQuota.",
+      "additionalProperties": {
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=7&length=20)

additionalProperties is not allowed:

[https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi032-only-use-additionalproperties-when-the-object-properties-are-dynamic-unknown-or-user-defined](https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi032-only-use-additionalproperties-when-the-object-properties-are-dynamic-unknown-or-user-defined)